ioctl/newcmd/config/config_test.go

```diff
@@ -28,6 +30,100 @@ func TestInitConfig(t *testing.T) {
 	require.Equal(filepath.Join(testPath, _defaultConfigFileName), cfgFilePath)
 }
 
+func TestConfigGet(t *testing.T) {
+	require := require.New(t)
+	tempDir := os.TempDir()
+	testPath, err := os.MkdirTemp(tempDir, "testCfg")
+	require.NoError(err)
+	defer func() {
+		testutil.CleanupPath(testPath)
+	}()
+	_configDir = testPath
+	cfg, cfgFilePath, err := InitConfig()
+	// the endpoint & default account are blank strings within the initial config, so I'm setting it here
+	cfg.Endpoint = "http://google.com"
+	cfg.DefaultAccount = config.Context{AddressOrAlias: "test"}
+	require.NoError(err)
+
+	info := newInfo(cfg, cfgFilePath)
+
+	tcs := []struct {
+		arg      string
+		expected string
+	}{
+		{
+			"endpoint",
+			"http://google.com    secure connect(TLS):true",
+		},
+		{
+			"wallet",
+			fmt.Sprintf("%s%s", tempDir, "testCfg"),
+		},
+		{
+			"defaultacc",
+			"{\n  \"addressOrAlias\": \"test\"\n}",
+		},
+		{
+			"explorer",
+			"iotexscan",
+		},
+		{
+			"language",
+			"English",
+		},
+		{
+			"nsv2height",
+			"5165641",
+		},
+		{
+			"analyserEndpoint",
+			"https://iotex-analyser-api-mainnet.chainanalytics.org",
+		},
+		{
+			"all",
+			"\"endpoint\": \"http://google.com\",\n  \"secureConnect\": true,\n  \"aliases\": {},\n  \"defaultAccount\": {\n    \"addressOrAlias\": \"test\"\n  },\n  \"explorer\": \"iotexscan\",\n  \"language\": \"English\",\n  \"nsv2height\": 5165641,\n  \"analyserEndpoint\": \"https://iotex-analyser-api-mainnet.chainanalytics.org\"\n}",
+		},
+	}
+
+	for _, tc := range tcs {
+		cfgItem, err := info.get(tc.arg)
+		require.NoError(err)
+		require.Contains(cfgItem, tc.expected)
```

Comment:
expected data should be set before checking if no error

Reply:
Would it be worth implementing the config set PR before this to use that functionality here?

Reply:
@huof6829  :)